### PR TITLE
Staging Cleanup

### PR DESCRIPTION
**Pull request by spuppala-rythmos**
Removed the testcases already in regression

changed the outputs which are not proper

updated the configs accordingly

### CODE REVIEW

File: tests/output/staging/10061-UNI-1355.cs [hunk 1]
```diff
@@ -8,9 +8,9 @@ public override MyNodeType NodeType
         {
             if (entity != null)
                 return entity.ReadState == ReadState.Initial ?
-                       source.NodeType :
-                       entity.EOF ? MyNodeType.EndEntity :
-                       entity.NodeType;
+                    source.NodeType :
+                    entity.EOF? MyNodeType.EndEntity:
```
Comment (mihaipopescu) [inline tests/output/staging/10061-UNI-1355.cs:12]: Why the space removal ?

**Review by mihaipopescu — APPROVED** [review]
Please add more comments why these changes are required.

File: tests/output/staging/10061-UNI-1355.cs (hunk 1 above)
Comment (spuppala-rythmos) [reply tests/output/staging/10061-UNI-1355.cs:12]: Previously there are two tabs and expected is one tab according to unity config and changes we have done. So removed one tab space

Comment (mihaipopescu) [reply tests/output/staging/10061-UNI-1355.cs:12]: I was talking about the space on the ternary operator `?`

**Review by hschlichter — APPROVED** [review]
Looks good. Builds are green.

**Pull request merged**